include/opc/ua/protocol/binary/common.h

```diff
@@ -50,7 +50,7 @@ namespace OpcUa
 
     enum ChunkType
     {
-      CHT_INVALID = 0,
+      CHT_INVALId = 0,
```

Comment:
Yes, I converted everything with sed and of course I got some mistakes :-)
I am going through and fixing...